Sources/CoreCommands/Options.swift

```diff
@@ -138,6 +138,10 @@ public struct LoggingOptions: ParsableArguments {
     /// The verbosity of informational output.
     @Flag(name: [.long, .customLong("vv")], help: "Increase verbosity to include debug output")
     public var veryVerbose: Bool = false
+
+    /// Whether logging output should be limited to error.
```

Comment:
nit: wording here is weird, unless it's meant to refer to an enum case, or however the corresponding logging level is declared.
```suggestion
    /// Whether logging output should be limited to `.error`.
```